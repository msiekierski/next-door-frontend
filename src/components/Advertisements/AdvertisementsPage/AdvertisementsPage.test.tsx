import React from "react";
import renderer from "react-test-renderer";
import AdvertisementPage from "./AdvertisementsPage";

it("renders correctly", () => {
  const tree = renderer.create(<AdvertisementPage />);
  expect(tree).toMatchSnapshot();
});
