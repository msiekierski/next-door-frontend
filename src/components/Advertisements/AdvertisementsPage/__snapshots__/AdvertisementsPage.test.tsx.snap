// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="d-flex justify-content-between mt-3 "
  >
    <a
      className="card-link"
      href="#"
      onClick={[Function]}
    >
      Create Advertisement
    </a>
  </div>
  Loading...
</div>
`;
